app/src/main/java/org/fossasia/openevent/app/contract/view/LoginView.java

```diff
@@ -0,0 +1,11 @@
+package org.fossasia.openevent.app.contract.view;
+
+public interface LoginView {
+
+    void showProgressBar(boolean show);
```

Comment:
@iamareebjamal I think `showProgressBar` will not be required by Presenter. It can be called by the view itself.